docker/dockerfiles/Dockerfile.server

```diff
@@ -19,9 +20,6 @@ RUN apt-get update; apt-get install -y \
     python3-software-properties \
     zip;
  
-### Without this Python thinks we're ASCII and unicode chars fail
```

Comment:
Don't think we need this since we're on Python 3 now.

Reply:
I remember needing this on Python 3, because otherwise non-ascii command line arguments would not be properly decoded. We could try removing this in a separate PR though.

Reply:
Added this back anyway, because it fails travis tests.